sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -464,7 +464,11 @@ def reader(self, test_bigquery_client=None):
 
 
 class BigQuerySink(dataflow_io.NativeSink):
-  """A sink based on a BigQuery table."""
+  """A sink based on a BigQuery table.
+
+  This BigQuery sink triggers a Dataflow native sink for BigQuery
+  that only supports batch pipelines.
```

Comment:
Can you also add following.

"Instead of using this sink directly, please use WriteToBigQuery transform that works for both batch and streaming pipelines."

Reply:
Done.